org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/servlet/ScanProcessTest.java

```diff
@@ -268,7 +268,7 @@ public void testScannableAndMonitor() throws Exception {
 		model.setExposureTime(0.001);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)